### PR TITLE
refactor: Migrate ContentViewModel to Typescript

### PR DESCRIPTION
**Pull request by thisisamir98**

### CODE REVIEW

File: src/script/view_model/content/PreferencesAVViewModel.ts [hunk 1]
```diff
@@ -196,7 +196,7 @@ export class PreferencesAVViewModel {
     await this.initiateDevices(mediaType);
   }
 
-  private async releaseDevices(mediaType: MediaType): Promise<void> {
+  public async releaseDevices(mediaType: MediaType): Promise<void> {
```
Comment (bennycode) [inline src/script/view_model/content/PreferencesAVViewModel.ts]: `public` is the default, so no need to declare it.

Comment (thisisamir98) [reply src/script/view_model/content/PreferencesAVViewModel.ts]: https://github.com/wireapp/wire-webapp/pull/9298/commits/8f08faae128071b7457ea81e5761fa7b5ef8336f

File: src/script/view_model/ContentViewModel.ts [hunk 2]
```diff
@@ -68,16 +148,13 @@ export class ContentViewModel {
...
-    this.switchPreviousContent = this.switchPreviousContent.bind(this);
-
+  constructor(mainViewModel: MainViewModel, repositories: Repositories) {
     this.elementId = 'center-column';
     this.mainViewModel = mainViewModel;
     this.conversationRepository = repositories.conversation;
     this.userRepository = repositories.user;
     this.logger = getLogger('ContentViewModel');
-    this.STATE = ContentViewModel.STATE;
+    this.State = ContentViewModel.STATE;
```
Comment (ffflorian) [inline src/script/view_model/ContentViewModel.ts:157]: Why not keep `STATE`?

Comment (thisisamir98) [reply src/script/view_model/ContentViewModel.ts:157]: typescript says property STATE already exists :(

Comment (ffflorian) [reply src/script/view_model/ContentViewModel.ts:157]: Hmm okay

**Pull request merged**